docs/technical-reference/running-modes.md

```diff
@@ -2,30 +2,30 @@
+This abstraction of a module template is used for generically deploying instances of a module within a Kyma Runtime at a specific Release Group we call `Channel` (for more information, visit the respective Chapter in the [Concept of Modularization](https://github.com/kyma-project/community/tree/main/concepts/modularization#release-channels)). It not only contains a specification of a Module with its underlying components through [OCM Component Descriptors](https://github.com/gardener/component-spec/blob/master/doc/proposal/02-component-descriptor.md), but also talks in detail about the schemas, labels and other essential resources.
 
-These serve as small-scale BOM's for all contents included in a module and can be interpreted by Lifecycle Manager and [Module Manager](https://github.com/kyma-project/module-manager/)
-to correctly install a module. (for more information, please have a look at the respective chapter in the [Kyma Modularization Concept](https://github.com/kyma-project/community/tree/main/concepts/modularization#component-descriptor))
+These serve as small-scale BoM's for all contents included in a module and can be interpreted by Lifecycle Manager and [Module Manager](https://github.com/kyma-project/module-manager/)
+to correctly install a module (for more information, please have a look at the respective chapter in the [Kyma Modularization Concept](https://github.com/kyma-project/community/tree/main/concepts/modularization#component-descriptor)).
 
 ## Versioning and Releasing
 
-Kyma up to Version 2.x was always a single release. However, the vision of lifecycle-manager is to fully encapsulate individual Modules, with each providing a (possibly fully independent) Release Cycle.
-However, Control-Plane deliveries are by design continuously shipped and improved. As a result, even if we will continue to support versioned Module Deliveries, the Lifecycle-Manager and its adjacent infrastructure will be maintained and delivered continously and it is recommended to track upstream as close as possible.
+Kyma up to Version 2.x was always a single release. However, the vision of Lifecycle Manager is to fully encapsulate individual Modules, with each providing a (possibly fully independent) Release Cycle.
+However, the Control-Plane deliveries are by design continuously shipped and improved. As a result, even if we continue to support versioned Module deliveries, the Lifecycle Manager and its adjacent infrastructure will be maintained and delivered continuously, and it is recommended to track upstream as close as possible.
```

Comment:
- In case you choose to drop the previous sentence, do we still need the "however"?
- "even if we continue to support versioned module deliveries" - we do that nowadays, don't we? Is this going to change? Or maybe I don't understand what this paragraph tries to tell me.
- You introduced the abbreviation "KCP" above; no need to spell it out anymore.

```suggestion
By design, the KCP deliveries are continuously shipped and improved. As a result, even if we continue to support versioned module deliveries, the Lifecycle Manager and its adjacent infrastructure will be maintained and delivered continuously, and it is recommended to track upstream as close as possible.
```

Reply:
1. Chose to keep the previous sentence.
2. I discussed with the team, it is indeed a confusing statement. I'm changing it to something more concrete. See my next commit.